### ISSUE TITLE
[Request] Label release binary as `Windows` instead of `PC`

### ISSUE THREAD

**Issue by kubark42** [LK8000/LK8000#1622]
In https://github.com/LK8000/LK8000/releases/tag/7.4.5, there is a binary labeled `PC`. Since Linux and macOS machines are also "personal computers", the intent would be clearer if the binary would use the word `Windows` to make it clear that binaries for the other OSes are not supported.

**Comment by brunotl** [comment]
linux is also supported but must be intalled from source...

**Comment by kubark42** [comment]
> linux is also supported but must be installed from source...

Good point, I updated my original post to make it clearer I am asking only about the released binaries.
